packages/desktop-client/src/components/common/Modal.tsx

```diff
@@ -311,6 +318,7 @@ export function ModalHeader({
         alignItems: 'center',
         position: 'relative',
         height: 60,
+        flex: 'none',
```

Comment:
This change is intentional: without it the ModalHeader shrinks, which looks bad on small viewports